docs/pages/custom-builds/schema.mdx

```diff
@@ -1392,6 +1392,97 @@ build:
   href="https://github.com/expo/eas-build/blob/main/packages/build-tools/src/steps/functions/startIosSimulator.ts"
 />
 
+#### `eas/send_slack_message`
+
+Sends a specified message to a configured Slack webhook URL, which then posts it in the related Slack channel.
+For more information on how to use Slack webhooks check [Slack documentation](https://api.slack.com/messaging/webhooks).
+
+```yaml send-slack-message.yml
+build:
+  name: Slack your team from custom build
+  steps:
+    # @info #
+    - eas/send_slack_message:
+        name: Send Slack message to a given webhook URL
+        inputs:
+          message: 'This is a message to plain input URL'
+          slack_hook_url: 'https://hooks.slack.com/services/[rest_of_hook_url]'
+    # @end #
+    # @info #
```

Comment:
There is not that much value in highlighting the whole actions code, especially without a descriptions.

Let's remove the highlight fully or retain the highlight on important action parts, preferably with tooltips for docs readers.

Reply:
I'll leave it one to indicate which is the described function and remove the rest